example/instance/vagrant/main.go

```diff
@@ -1,27 +1,29 @@
 package main
 
 import (
+	"os"
+
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/infrakit/cli"
 	"github.com/docker/infrakit/plugin/instance/vagrant"
 	instance_plugin "github.com/docker/infrakit/spi/http/instance"
 	"github.com/spf13/cobra"
-	"os"
 )
 
 func main() {
 
 	var name string
 	var logLevel int
 	var dir string
+	var template string
 
 	cmd := &cobra.Command{
 		Use:   os.Args[0],
 		Short: "Vagrant instance plugin",
 		Run: func(c *cobra.Command, args []string) {
 
 			cli.SetLogLevel(logLevel)
-			cli.RunPlugin(name, instance_plugin.PluginServer(vagrant.NewVagrantPlugin(dir)))
+			cli.RunPlugin(name, instance_plugin.PluginServer(vagrant.NewVagrantPlugin(dir, template)))
```

Comment:
I was thinking `string`, but `*Template` is even better since you can then fail fast on a bogus template. :+1: